root/utility/tableColumns.js

```diff
@@ -404,11 +404,11 @@ export function defineReleaseLabelsColumn(
 
 export function defineSeriesNumberColumn(
   props: {
-    +seriesItemNumbers: {+[entityId: number]: string},
+    +seriesItemNumbers: $ReadOnlyArray<string>,
   },
 ): ColumnOptions<CoreEntityT, number> {
   return {
-    Cell: ({cell: {value}}) => props.seriesItemNumbers[value],
+    Cell: ({row: {index}}) => props.seriesItemNumbers[index + 1],
```

Comment:
Aren't both `seriesItemNumbers` and the row index 0-based? How come the + 1 there?

Reply:
Because I was initializing it as `my @item_numbers = [];` rather than just `my @item_numbers;`. Whoops!. Fixed.